python/orca/src/bigdl/orca/learn/openvino/estimator.py

```diff
@@ -224,31 +227,28 @@ def predict_transform(dict_data, batch_size):
                                   "x in each shard should be a ndarray or a list of ndarray.")
             return feature_data
 
+        def update_result_shard(data):
```

Comment:
I think it is better to update in add_predict_to_pd_xshards so that it can be used for other estimators as well?
Also it is not general to always convert to list? Add some check if it is ndarray then convert to list?